src/components/CreateCourse/CreateCourse.jsx

```diff
@@ -34,6 +36,19 @@ function CreateCourse() {
 
   const ALERT_TEXT = getAlertText();
 
+  useEffect(() => {
+    async function fetchAndCombineAuthors() {
+      const apiAuthors = await getAuthorsAPI();
+      const localStorageAuthors =
```

Comment:
Why would we save authors into local storage? This is dynamic data that is fetched from API and doesn't need to be saved into persistent browser storage